modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -78,17 +56,39 @@ const getDelimiter = (dateFormat) => {
 	return dateDelimiter;
 };
 
+const getLocaleDateFormat = (format = 'L') => {
+	return moment.localeData().longDateFormat(format);
+};
+
+const getMaskByDateFormat = (format) => {
+	const mask = [];
+
+	for (let i = 0; i < format.length; i++) {
+		if (/[a-z]/i.test(format[i])) {
+			mask.push(/\d/);
```

Comment:
@alinedoleron, can you please add the regular expressions (`/[a-z]/i` and `/\d/`) to constants to let the code a bit clear?

Besides, is this code the same used on Workflow? If so, maybe you and the Workflow guys should consider create a util to share this code among different modules. What the Workflow guys think about it?